clar2wasm/src/tools.rs

```diff
@@ -246,6 +248,19 @@ impl Default for TestEnvironment {
     }
 }
 
+pub fn execute<F, T, E>(conn: &mut ClarityDatabase, f: F) -> std::result::Result<T, E>
```

Comment:
Why was this moved to a standalone function when it's clearly coupled to `ClarityDatabase`?